eva/parser/evaql/evaql_parser.g4

```diff
@@ -55,7 +55,7 @@ createDatabase
 createIndex
     : CREATE
       INDEX uid indexType?
```

Comment:
How about if we move `USING HNSW` to the end? 
`CREATE INDEX name ON tableName(feature) USING HNSW`?